tests/test_box.py

```diff
@@ -401,3 +407,43 @@ def fn(a, b):
         fn(1)
 
     excinfo.match('b')
+
+
+def test_chainbox_put_changes_box():
+    testbox = picobox.Box()
+    testchainbox = picobox.ChainBox(testbox)
+
+    with pytest.raises(KeyError):
+        testchainbox.get('the-key')
+    testchainbox.put('the-key', 42)
+
+    assert testbox.get('the-key', 42)
```

Comment:
Hmm, do you really want to pass the value of `default` argument here? I thought, the point of the test was to check that `testbox` has been modified and now contains `the-key`.

Reply:
Damn copy-paste. :( You're absolutely right.